README.md

```diff
@@ -1,4 +1,22 @@
-# Arcus Template
-Template repository for new components in Arcus ecosystem.
+# Arcus - Testing
+[![Build Status](https://dev.azure.com/codit/Arcus/_apis/build/status/Commit%20builds/CI%20-%20Arcus.Testing?branchName=master)](https://dev.azure.com/codit/Arcus/_build/latest?definitionId=804&branchName=master)
+[![NuGet Badge](https://buildstats.info/nuget/Arcus.Testing.Logging?includePreReleases=true)](https://www.nuget.org/packages/Arcus.Testing.Logging/)
 
-:warning: In order to finalize the setup, don't forget to run out [GitHub Repo Configurator](https://github.com/arcus-azure/arcus/tree/master/tools/github-repo-configurator).
+Reusable testing components for Arcus repo's.
+
+![Arcus](https://raw.githubusercontent.com/arcus-azure/arcus/master/media/arcus.png)
+
+# Installation
+Soon you will be able to install it via NuGet:
+
+```shell
+PM > Install-Package Arcus.Testing.Logging
+```
+
+# Documentation
+All documentation can be found on [here](https://testing.arcus-azure.net/).
```

Comment:
Actually I'm wondering if we need to have this since it's for us internally only?

Maybe easier if we just shove those docs in this README - Thougths?

Reply:
Hmm, don't know. We don't know how this repo may evolve and the amount of source code it will contain. An over-sized README is maybe not that clear what the repo actually contains.

I think indeed, that we may don't need a public site, but... a GitHub wiki instead? Hmm, not sure. I guess the things here _could_ be used in other projects outside Arcus too.

Reply:
Wikis are terrible because it's a seperate git repo to PR to, I prefer to keep it in the repo.

Let's move the list of features in the README and point to dedicated feature files under `docs/<feature-name>.md` so that README is not polluted.

A good middle ground?